cli/native/src/main/scala/com/lightbend/rp/reactivecli/process/NativeProcess.scala

```diff
@@ -22,10 +22,15 @@ import scalanative.native._
 import slogging._
 
 object NativeProcess extends LazyLogging {
-  def exec(args: String*): Future[(Int, String)] =
+  def exec(args: Seq[String], stdinFile: Option[String] = None): Future[(Int, String)] =
     withTempFile { outputFile =>
       Zone { implicit z =>
-        val code = stdlib.system(toCString(s"${command(args)} > $outputFile 2>&1"))
+        val cmd = if (stdinFile.isDefined)
+          s"${command(args)} < ${stdinFile.get} > $outputFile 2>&1"
+        else
+          s"${command(args)} > $outputFile 2>&1"
+
```

Comment:
I think we should technically be escaping the `stdinFile` and `outputFile`. In our immediate case we're fine since we're only passing in temp files, but if the method was used for something else that contained spaces it would break. Note this was an issue before this PR as well.

Would you be able to file an issue so we can follow up later?

Reply:
Of course - https://github.com/lightbend/reactive-cli/issues/109